### DISCUSSION TITLE
Where is located the config file and how do I change the ssl port to the standard port (TCP-443) ?

### DISCUSSION THREAD

**Discussion by transarmy** [requarks/wiki#3418] [Help / Questions]
**Question**
Where is located the config file and how do I change the ssl port to the standard port (TCP-443) ?

**Host Info (please complete the following information):**
OS: AWS
Wiki.js version: Latest
Database engine: Default

**Comment by NGPixel** [comment]
Have you [read the docs](https://docs.requarks.io/install/aws)?

**Comment by 3DSkaoi** [comment]
You also have the ability to redirect port.

**Comment by GZangl** [comment]
As you are using AWS, you could use a Load Balancer together with an ACM certificate. You need to define a listener in the Load Balancer and also a Target Group (this is your http address) to forward https.
